reference/content/whats-new.md

```diff
@@ -9,11 +9,11 @@ title = "What's New"
 
 # What's new in 4.6
 
-* Refer to [What's New - Java Sync Documentation](https://www.mongodb.com/drivers/java/sync/v4.6) for more information.
+* Refer to [What's New - Java Sync Documentation](https://www.mongodb.com/docs/drivers/java/sync/v4.6) for more information.
```

Comment:
Are these links supposed to go to sections on [this What's New page](https://www.mongodb.com/docs/drivers/java/sync/v4.6/whats-new/), or are you just linking to the Java driver docs in general? I think it makes sense to link to the specific What's New page (but if it doesn't, feel free to ignore this)

Reply:
That's a good call. I think originally they just pointed to the docs home page but since we are specifying "What's New" I think it makes sense to link directly there!